docs/templates.md

```diff
@@ -348,59 +348,62 @@ Most types can be implicitly converted to `Template`. No methods are defined.
+- `.before(date: String) -> Boolean`: True if the timestamp is before, but not including, the given date.
 
 ### TimestampRange type
 
 The following methods are defined.
 
-* `.start() -> Timestamp`
-* `.end() -> Timestamp`
-* `.duration() -> String`
+- `.start() -> Timestamp`
+- `.end() -> Timestamp`
+- `.duration() -> String`
 
 ### TreeDiff type
 
 This type cannot be printed. The following methods are defined.
 
-* `.files() -> List<TreeDiffEntry>`: Changed files.
-* `.color_words([context: Integer]) -> Template`: Format as a word-level diff
+- `.files() -> List<TreeDiffEntry>`: Changed files.
+- `.color_words([context: Integer]) -> Template`: Format as a word-level diff
   with changes indicated only by color.
-* `.git([context: Integer]) -> Template`: Format as a Git diff.
-* `.stat([width: Integer]) -> DiffStats`: Calculate stats of changed lines.
-* `.summary() -> Template`: Format as a list of status code and path pairs.
+- `.git([context: Integer]) -> Template`: Format as a Git diff.
+- `.stat([width: Integer]) -> DiffStats`: Calculate stats of changed lines.
+- `.summary() -> Template`: Format as a list of status code and path pairs.
 
 ### TreeDiffEntry type
 
 This type cannot be printed. The following methods are defined.
 
-* `.path() -> RepoPath`: Path to the entry. If the entry is a copy/rename, this
+- `.path() -> RepoPath`: Path to the entry. If the entry is a copy/rename, this
   points to the target (or right) entry.
-* `.status() -> String`: One of `"modified"`, `"added"`, `"removed"`,
+- `.status() -> String`: One of `"modified"`, `"added"`, `"removed"`,
   `"copied"`, or `"renamed"`.
-* `.source() -> TreeEntry`: The source (or left) entry.
-* `.target() -> TreeEntry`: The target (or right) entry.
+- `.source() -> TreeEntry`: The source (or left) entry.
+- `.target() -> TreeEntry`: The target (or right) entry.
 
 ### TreeEntry type
 
 This type cannot be printed. The following methods are defined.
 
-* `.path() -> RepoPath`: Path to the entry.
-* `.conflict() -> Boolean`: True if the entry is a merge conflict.
-* `.file_type() -> String`: One of `"file"`, `"symlink"`, `"tree"`,
+- `.path() -> RepoPath`: Path to the entry.
+- `.conflict() -> Boolean`: True if the entry is a merge conflict.
+- `.file_type() -> String`: One of `"file"`, `"symlink"`, `"tree"`,
   `"git-submodule"`, or `"conflict"`.
-* `.executable() -> Boolean`: True if the entry is an executable file.
+- `.executable() -> Boolean`: True if the entry is an executable file.
 
 ## Configuration
 
 The default templates and aliases() are defined in the `[templates]` and
 `[template-aliases]` sections of the config respectively. The exact definitions
-can be seen in the `cli/src/config/templates.toml` file in jj's source tree.
+can be seen in the [`cli/src/config/templates.toml`][1] file in jj's source
+tree.
+
+[1]: https://github.com/jj-vcs/jj/blob/main/cli/src/config/templates.toml
```

Comment:
Nit: you can inline the notation by using ```[`cli/src/config/templates.toml`]: <link>``` which is imo simpler.

Reply:
I think you can revert to this version, sorry for my miss-guidance here.

Reply:
Done. I meant to check that it was rendering right and forgot; sorry.